sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubProducerClient.java

```diff
@@ -226,7 +223,7 @@ public String getFullyQualifiedNamespace() {
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
     public EventHubProperties getEventHubProperties() {
-        return producer.getEventHubProperties().block(tryTimeout);
+        return producer.getEventHubProperties().block();
```

Comment:
To make sure I understand correctly, inside the async client, there is already a tryTimeout applied when performing this operation, so it's duplicated?